frontend/src/App.jsx

```diff
@@ -109,6 +141,20 @@ function App() {
         </Routes>
       </main>
       <Footer />
+      {isMobile && (
+         <ActionModal
+         isOpen={modal}
+         title="Mobile website restriction"
+         subTitle=""
+         content={[
```

Comment:
please, put the text into sub-title. not content

Reply:
done